src/sonic-ctrmgrd/ctrmgr/container

```diff
@@ -0,0 +1,367 @@
+#!/usr/bin/python3 -u
+
+import argparse
+import docker
+import inspect
+import json
+import os
+import sys
+import syslog
+import time
+
+from swsscommon import swsscommon
+
+sys.path.append('/usr/share/sonic/scripts/')
+import container_state
+
+state_db = None
+
+# DB field names
+FEATURE_TABLE = "FEATURE"
+SET_OWNER = "set_owner"
+NO_FALLBACK = "no_fallback_to_local"
+
+CURRENT_OWNER = "current_owner"
+UPD_TIMESTAMP = "update_time"
+DOCKER_ID = "container_id"
```

Comment:
why not use CONTAINER_ID to match the string?

Reply:
Can be done. A local var only. Will do.

Reply:
done